src/mulsi/reader.py

```diff
@@ -0,0 +1,31 @@
+"""Module for reading representations from a model.
+"""
+
+from abc import ABC, abstractmethod
+from typing import Dict, Type
+
+
+class RepresentationReader(ABC):
```

Comment:
So this class will save representation TensorDicts for a given model ?

Reply:
Nope, that's the representation. This is for deducting concepts or whatever.